code/game/area/Space Station 13 areas.dm

```diff
@@ -1222,7 +1222,7 @@ proc/process_adminbus_teleport_locs()
 	icon_state = "showroom"
 
 /area/assembly/assembly_line //Derelict Assembly Line
```

Comment:
Alright you've got some problems here
First of all, this area is unpowered
Secnod, it's still called `assembly_line`
Third, why not just make a new area rather than just changing the name of an  old one?